.github/workflows/shared-run-e2e-tests.yaml

```diff
@@ -0,0 +1,141 @@
+name: Run operator E2E tests
+on:
+  workflow_call:
+    inputs:
+      core_root:
+        description: 'The root path for uid2-core folder'
+        type: string
+        default: '../uid2-core'
+      optout_root:
+        description: 'The root path for uid2-optout folder'
+        type: string
+        default: '../uid2-optout'
+      admin_root:
+        description: 'The root path for uid2-admin folder'
+        type: string
+        default: '../uid2-admin'
+      operator_root:
+        description: 'The root path for uid2-operator folder'
+        type: string
+        default: '../uid2-operator'
+      operator_image_version:
+        description: 'The version of UID2 operator image'
+        type: string
+        default: 'latest'
+      core_image_version:
+        description: 'The version of UID2 core image'
+        type: string
+        default: '2.15.8-d2e5469826-default'
+      optout_image_version:
+        description: 'The version of UID2 optout image'
+        type: string
+        default: '2.7.3-4f813a8ad9-default'
+      e2e_image_version:
+        description: 'The version of E2E image'
+        type: string
+        default: 'latest'
+      operator_type:
+        description: 'The type of operator [either public or private]'
+        type: string
+        default: 'public'
+      uid2_e2e_identity_scope:
+        description: 'Environment variable to run the E2E test'
+        required: false
+        type: string
+        default: 'UID2'
+      uid2_e2e_pipeline_operator_type:
+        description: 'Environment variable to run the E2E test'
+        required: false
+        type: string
+        default: 'PUBLIC'
+      uid2_e2e_pipeline_operator_url:
+        description: 'Environment variable to run the E2E test'
+        required: false
+        type: string
+        default: 'http://publicoperator:8080'
+      uid2_e2e_pipeline_operator_cloud_provider:
+        description: 'Environment variable to run the E2E test'
+        required: false
+        type: string
+        default: 'PUBLIC'
+
+env:
+  REGISTRY: ghcr.io
+
+jobs:
+  e2e-test:
+    name: E2E Test
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      packages: read
+      id-token: write
+    steps:
+      - name: Log in to the Docker container registry
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          # we use personal access token here since we need to pull images from private repo (core)
+          password: ${{ secrets.GHCR_PAT }}
```

Comment:
Do we still need this?

Same question below with anywhere `GHCR_PAT` is passed

Reply:
Good point, removed